applications/CoSimulationApplication/custom_external_libraries/CoSimIO/external_libraries/asio/include/asio/error.hpp

```diff
@@ -66,7 +66,7 @@ namespace error {
 enum basic_errors
 {
   /// Permission denied.
-  access_denied = ASIO_SOCKET_ERROR(EACCES),
+  access_denied = ASIO_SOCKET_ERROR(EACCESS),
```

Comment:
I dont think this can be changed, it comes from an external lib

Reply:
reverted